lib/CXGN/Stock.pm

```diff
@@ -309,9 +309,9 @@ sub store {
             $self->_update_stockprop('organization', $self->organization_name());
         }
         if ($self->population_name){
-	    print STDERR "**STOCK.PM This stock has population name " . $self->population_name . "\n\n";
+            print STDERR "**STOCK.PM This stock has population name " . $self->population_name . "\n\n";
             #DO NOT INSERT POPULATION RELATIONSHIP FROM THE STOCK STORE FUNCTION
-            ##$self->_update_population_relationship();
+            $self->_update_population_relationship();
```

Comment:
move this out of the store function in Stock.pm 
Call the update from AddPopulation